apps/wallet-mobile/src/components/BottomSheetModal/BottomSheetModal.tsx

```diff
@@ -1,25 +1,29 @@
 import BottomSheet, {BottomSheetBackdrop} from '@gorhom/bottom-sheet'
 import {useNavigation} from '@react-navigation/native'
-import React, {useCallback, useEffect, useRef, useState} from 'react'
-import {Modal as RNModal, StyleSheet, Text, View} from 'react-native'
+import React from 'react'
+import {Modal as RNModal, StyleSheet, Text, TouchableOpacity, View, ViewStyle} from 'react-native'
+
+import {Icon} from '../Icon'
+import {Spacer} from '../Spacer'
 
 type BottomSheetProps = {
```

Comment:
or only `Props` for the exported component. 
```suggestion
type BottomSheetModalProps = {
```